### DISCUSSION TITLE
CCU Feld zum Update Datei Hochladen fehlt

### DISCUSSION THREAD

**Discussion by Tischi96** [OpenCCU/OpenCCU#2029] [🇩🇪 Fragen&Antworten] [answered]
Ich würde gerne meine CCU Updaten aber mir fehlt das Feld um die Datei hochzuladen. 
Mir wird nur das Feld angezeigt um die Entwickler zu unterstützen. Hier passiert aber auch nichts.
Was kann ich tun?

![2022-11-01 13_00_52-Window](https://user-images.githubusercontent.com/117171819/199228057-3b692f88-ee7d-4fe2-a335-91b5cc5f4130.png)

**Comment by jens-maus** [comment, marked as answer]
Das hier ist das GitHub Projekt von "RaspberryMatic". Ganz offensichtlich nutzt du jedoch "piVCCU". Insofern geht es für dich nun hier weiter, da "RaspberryMatic" und "piVCCU" zwei Paar Schuhe sind:

https://github.com/alexreinert/piVCCU
